de-de/dhall-de.html.markdown

```diff
@@ -0,0 +1,378 @@
+---
+language: Dhall
+translators:
```

Comment:
Please add the contributors field from the original dhall doc here.

Reply:
Done, see force push.